src/mlpack/methods/decision_tree/decision_tree_impl.hpp

```diff
@@ -430,6 +436,7 @@ void DecisionTree<FitnessFunction,
                                       const size_t numClasses,
                                       WeightsType&& weights,
                                       const size_t minimumLeafSize,
+                                      const double minimumGainSplit = 1e-7,
```

Comment:
I think it's not actually legal to set a default parameter when it's already been specified in the declaration, so no need for the `= 1e-7` here.